tests/fileHandling/reader/FileReaderTest.cpp

```diff
@@ -0,0 +1,154 @@
+//
+// Created by daniel on 11.05.24.
+//
+
+#include <gtest/gtest.h>
+#include "fileHandling/reader/FileReader.h"
+#include <spdlog/spdlog.h>
+
+/**
+ * Set up test environment for first test suite.
+ */
+class FileReaderTest : public testing::Test {
+protected:
+    ParticleContainer pc;
+
+    void SetUp() override {
+        //Deactivate all console output
+        spdlog::set_level(spdlog::level::off);
+    }
+};
+
+/**
+ * Is the file read correctly?
```

Comment:
This is just nitpicking but I would rephrase it a bit and say something like "This test checks if the <> file is read correctly" or something similar